lms/templates/learner_dashboard/program_details_view.underscore

```diff
@@ -1,7 +1,7 @@
 <header class="program-header">
     <div>
-        <h2 class="hd-2">Program Title</h2>
-        <p>Program Subtitle</p>
+        <h2 class="hd-2"><%- gettext(name) %></h2>
+        <p><%- gettext(subtitle) %></p>
```

Comment:
This is the section I am working on with https://github.com/edx/edx-platform/pull/12550, please revert your changes to avoid PR merge confusion.